include/openthread/link.h

```diff
@@ -95,6 +95,17 @@ typedef enum otMacFilterAddressMode
     OT_MAC_FILTER_ADDRESS_MODE_DENYLIST,  ///< Denylist address filter mode is enabled.
 } otMacFilterAddressMode;
 
+/**
+ * Defines SSED-to-SSED mode.
+ *
+ */
+typedef enum otMacSsedToSsedMode
+{
+    OT_MAC_SSED_TO_SSED_MODE_CENTER, ///< SSED-to-SSED center mode.
+    OT_MAC_SSED_TO_SSED_MODE_END,    ///< SSED-to-SSED end mode.
+    OT_MAC_SSED_TO_SSED_MODE_NONE,   ///< Default mode. Won't have SSED-to-SSED behavior.
+} otMacSsedToSsedMode;
```

Comment:
```suggestion
typedef enum otMacSsedToSsedMode
{
    OT_MAC_SSED_TO_SSED_MODE_DISABLED = 0,   ///< Default mode. Disable SSED-to-SSED link.
    OT_MAC_SSED_TO_SSED_MODE_CENTER = 1, ///< SSED-to-SSED center mode.
    OT_MAC_SSED_TO_SSED_MODE_END = 2,    ///< SSED-to-SSED end mode.
} otMacSsedToSsedMode;
```

Change the order and put disabled first. How about putting this to include/openthread/thread.h like `otDeviceRole`?

Reply:
I think it also makes sense. But I think we should either put all these things into `link.h` or put all of them into `thread.h`. Since most CSL related things have been put into `link.h`, I prefer to put them in `link.h`.